formater/boomr.csv

```diff
@@ -209,8 +209,6 @@ Oppdatert per 1.1.2022;;;;;;;;;;
 4617;4617 Kvinnherad;;;12;Fonna;;;3;Helse Vest RHF;
 4618;4618 Ullensvang;;;12;Fonna;;;3;Helse Vest RHF;
 4619;4619 Eidfjord;;;12;Fonna;;;3;Helse Vest RHF;
-1101;1101 Eigersund;;;13;Stavanger;;;3;Helse Vest RHF;
-1103;1103 Stavanger;;;13;Stavanger;;;3;Helse Vest RHF;
 1103;1103 Stavanger;110301;01 Hundv�g;13;Stavanger;;;3;Helse Vest RHF;
```

Comment:
Er det noe poeng å flytte disse to linjene?

Reply:
Neppe..er nok fordi jeg sorterte filen før jeg lagret, og sorteringen ble ulik den tidligere.

Reply:
Nå er det sortert en gang til. Fremdeles ikke helt som før, men det må du nok leve med! :)

Reply:
Nydelig sortert!